lara-ecs.yml

```diff
@@ -399,10 +384,7 @@ Resources:
           - - !If [ AddFakeAuthProviderCond, HAS_STAGING, !Ref 'AWS::NoValue']
```

Comment:
@scytacki I'm not sure should update the `# fake auth provider config` section to use something else besides HAS_STAGING.  Any ideas?

Reply:
There was a problem when there was only a single auth provider, so the fake one was added to hack around the problem. I don't remember if we tried to fix it and gave up, or were just stressed trying to release quickly. My guess is the latter. 

So I'd go ahead and remove it. If the problem is still there it will only show up on single provider portals. And I'd guess we have the resources to fix the actual problem if it is still there.